.github/dependabot.yml

```
@@ -0,0 +1,10 @@
+version: 2
+updates:
+  - package-ecosystem: npm
+    directory: '/'
+    schedule:
+      interval: monthly
+    open-pull-requests-limit: 5
+    versioning-strategy: widen
```

[note] This project is an *application*, so I think `increase` is more proper than `widen`:

```suggestion
    versioning-strategy: increase
```

See also: https://docs.github.com/en/github/administering-a-repository/configuration-options-for-dependency-updates#versioning-strategy